members/CusZc2ffb85b1496334264.yaml

```diff
@@ -34,6 +34,8 @@ contact_form:
         required: true
     - click_on:
       - selector: div.GIY1LSJISC button.primary
+    - wait:
+      - value: 3
     - find:
```

Comment:
there are four yamls pointed at this same form - could you update those as well?